src/tsickle.ts

```diff
@@ -1461,18 +1461,33 @@ class Annotator extends ClosureRewriter {
    * Processes an EnumDeclaration into a Closure type. Always emits a Closure type, even in untyped
    * mode, as that should be harmless (it only ever uses the number type).
    */
-  private maybeProcessEnum(node: ts.EnumDeclaration) {
-    // Gather the members of enum, saving the constant value or
-    // initializer expression in the case of a non-constant value.
-    const members = new Map<string, number|ts.Node>();
-    let i = 0;
+  private processEnum(node: ts.EnumDeclaration) {
+    // Emit the enum declaration, which looks like:
+    //   /** @enum {number} */
+    //   const Foo = {BAR: 0, BAZ: 1, ...};
+    //   export {Foo};  // even if originally exported on one line.
+    // This declares an enum type for Closure Compiler (and Closure JS users of this TS code).
+    // Splitting the enum into declaration and export is required so that local references to the
+    // type resolve ("@type {Foo}").
+    this.emit('\n');
+    const name = node.name.getText();
+
+    this.emit(`/** @enum {number} */\n`);
+    this.emit(`const ${name}: any = {\n`);
```

Comment:
We don't type check the output, right?  So you can drop the ":any".  Elsewhere in tsickle I remember using some ": DontTypeCheckMe" annotation to make this explicit.